addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1539,4 +1540,105 @@ options.registry.anchorName = options.Class.extend({
         this.$target.trigger('content_changed');
     },
 });
+
+WysiwygMultizone.include({
```

Comment:
Why not making that in the actual definition of the `WysiwygMultizone` widget instead of doing an include inside the same app ?